CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## 2024-01-25 v1.23.0
```

Comment:
We've missed this date for an official v1.23.0 version. Could you leave the date blank for now?

```suggestion
## Upcoming
```

Otherwise, if you skip any changes to the CHANGELOG.md, our staff will write the changelog with the next tagged release.

Reply:
Done. This was just intended as a placeholder.